### PR TITLE
Exclude SLE12 platform.

### PR DESCRIPTION
**Pull request by aplanas**
Depends on crowbar/barclamp-crowbar#1175

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

**Comment by tboerger** [comment]
LGTM

File: crowbar_framework/app/models/trove_service.rb [hunk 1]
```diff
@@ -25,7 +25,10 @@ def role_constraints
       {
         "trove" => {
           "unique" => false,
-          "count" => -1
+          "count" => -1,
```
Comment (vuntz) [inline crowbar_framework/app/models/trove_service.rb]: Not really related to this PR, but I'm unsure that the count constraint is correct :/

**Comment by aplanas** [comment]
* Exclude Widnows platforms

**Comment by aplanas** [comment]
* Set `count` to 1

**Pull request merged**